lib/puppet/provider/ios_ntp_access_group/command.yaml

```diff
@@ -0,0 +1,17 @@
+---
+get_values:
+  default: 'show running-config | include ^ntp access-group'
+get_instances:
+  default: 'ntp access-group.+\\n'
+set_values:
+  default: 'ntp access-group <access_group_type> <name>'
```

Comment:
From a quick test, on some devices (mainly the 6503) the command is just `ntp access-group <name>` as `ipv6` seems to be implied by the access list and no need to specify ipv6 would be my guess.

Probably best to verify what other devices differ and have something like https://github.com/puppetlabs/cisco_ios/blob/master/lib/puppet/provider/snmp_notification_receiver/command.yaml#L8-L10

i.e. 

```
set_values:
  default: 'ntp access-group <access_group_type> <name>'
  '6503': 'ntp access-group <name>'
```

Then put an exclusion on `ipv6_access_group` for the devices which don't support setting it as `ipv6`.